javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkPerformanceReporter.java

```diff
@@ -5,55 +5,51 @@
  */
 package javaslang.benchmark;
 
-import javaslang.Function2;
-import javaslang.Tuple;
-import javaslang.Tuple2;
-import javaslang.collection.Array;
-import javaslang.collection.CharSeq;
+import javaslang.*;
+import javaslang.collection.*;
 import javaslang.collection.List;
 import javaslang.collection.Map;
-import javaslang.collection.TreeMap;
 import javaslang.collection.Vector;
 import javaslang.control.Option;
 import org.openjdk.jmh.infra.BenchmarkParams;
-import org.openjdk.jmh.results.BenchmarkResult;
-import org.openjdk.jmh.results.Result;
-import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.results.*;
 import org.openjdk.jmh.util.ListStatistics;
 
 import java.text.DecimalFormat;
-import java.util.Collection;
-import java.util.Comparator;
+import java.util.*;
 import java.util.stream.Collectors;
 
 public class BenchmarkPerformanceReporter {
     private static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("#,##0.000");
     private static final DecimalFormat PERFORMANCE_FORMAT = new DecimalFormat("#0.00");
     private static final DecimalFormat PCT_FORMAT = new DecimalFormat("0.00%");
+    private final Class<?> benchmarkClass;
     private final Collection<RunResult> runResults;
     private final String targetImplementation;
     private final double outlierLowPct;
     private final double outlierHighPct;
 
-    public static BenchmarkPerformanceReporter of(Collection<RunResult> runResults) {
-        return of(runResults, "slang", 0.3, 0.1);
+    public static BenchmarkPerformanceReporter of(Class<?> benchmarkClass, Collection<RunResult> runResults) {
+        return of(benchmarkClass, runResults, "slang", 0.3, 0.1);
```

Comment:
It would be best if the display order would be the same as the order the groups were defined in, not alphabetically